packages/devtools_app/lib/src/service/editor/api_classes.dart

```diff
@@ -396,3 +425,34 @@ class EditorDevice with Serializable {
         Field.supported: supported,
       };
 }
+
+/// UI settings for an editor's theme.
+class EditorTheme with Serializable {
```

Comment:
can this whole class be turned into an extension type instead?

see this example: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/screens/logging/logging_controller.dart/#L626

Reply:
fine to leave as is for consistency